### ISSUE TITLE
MITM DNS false (I hope) report?

### ISSUE THREAD

**Issue by ArtOfCombat** [Lunatrius/LunatriusCore#30]
My Antivirus reported a MITM DNS hijack in your code. I hope you are not doing this intentionally, and if you aren't, please check why it might be falsely reported.

**Comment by bspkrs** [comment]
Was it LunatriusCore specifically that produced the report? Which antivirus vendor, software, and version? Can you provide the specific details of the virus report?

**Comment by ArtOfCombat** [comment]
I will shortly - it was LC. It was in Avast, will get version too.

**Comment by Lunatrius** [comment]
Turn off update checking in the settings. I have added invisible updates for the meanwhile mentioning this. I've been trying around to get the mods updated but I haven't really found enough time to start migrating the mods.

**Comment by Lunatrius** [comment]
Turns out that the nameserver domain expired for the web server where I'm hosting my stuff. I've fixed up the DNS records and it should not happen anymore.

With all that said, everyone should still turn off update checks. I'm not sure when/if I'll be able to update my mods, so there is no point in having the mod check for updates.

In hindsight, I should've added some sort of "kill switch", to automatically turn off update checks for old/outdated versions.

**Comment by Lunatrius** [comment]
tl;dr: the worst thing that could have happened, would be the MITM AV warning, the client side does does not execute anything that is fetched from the server (the [/json](https://github.com/Lunatrius/LunatriusCore/blob/f2a1636c22002c9857e6dd9d1d71feaa77b0948c/src/main/java/com/github/lunatrius/core/version/VersionChecker.java#L51-L58) endpoint)
